mender-convert-modify

```diff
@@ -277,6 +222,61 @@ ${data_part_device}   /data          auto       ${MENDER_DATA_PART_FSTAB_OPTS}
 $fstab
 EOF
 
+log_info "Installing Mender client and related files"
+
+if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
+    case "${MENDER_CLIENT_VERSION}" in
+        1.* | 2.* | 3.*)
+            log_info "Installing Mender client version ${MENDER_CLIENT_VERSION}"
+            deb_get_and_install_package mender-client mender-client "${MENDER_CLIENT_VERSION}"
+            mender_client_deb_name="${DEB_NAME}"
+            ;;
+        *)
+            # For 4.x, master and latest(*) install the new packages
+            # At the time of writing, this will be broken for "latest" until mender-client v4 is
+            # released. Having an special handling deemed to complex for the short time benefit.
+            log_info "Installing Mender Auth and Mender Update version ${MENDER_CLIENT_VERSION}"
+            deb_get_and_install_package mender-client mender-auth "${MENDER_CLIENT_VERSION}"
+            deb_get_and_install_package mender-client mender-update "${MENDER_CLIENT_VERSION}"
+            mender_client_deb_name="${DEB_NAME}"
+
+            log_info "Installing Mender Flash version ${MENDER_FLASH_VERSION}"
+            deb_get_and_install_package mender-flash mender-flash "${MENDER_FLASH_VERSION}"
+            ;;
+    esac
+
+    # Save installed client version for tests in Yocto variable format
+    testscfg_add "PREFERRED_VERSION:mender-client" "$(echo ${mender_client_deb_name} | sed -r 's/.*_([0-9]+\.[0-9]+\.[0-9]+).*/\1/')"
+fi
+
+if [ "${MENDER_ENABLE_SYSTEMD}" == "y" ]; then
+    case "${MENDER_CLIENT_VERSION}" in
+        1.* | 2.* | 3.*)
+            run_and_log_cmd "sudo ln -sf /lib/systemd/system/mender-client.service \
```

Comment:
Hmm, are the packages not enabled by default? :thinking:

Reply:
Hmm, this might not be necessary anymore. I'll check it.

Reply:
I also assume that these should be now handled by `dpkg` in postinst scripts.

Reply:
It is. I removed this part.